src/app/dashboard/dashboard.html

```diff
@@ -1,33 +1,26 @@
+<<<<<<< HEAD
+<div id="dashboardContainer">
+
+	<header id="dashboardHeader">
```

Comment:
why do you have both a header and a nav bar?
they are completely different concepts
if you plan on having a fixed nav bar, you might want to stray away from the header